core/config.py

```
@@ -107,42 +89,40 @@ def from_env() -> "AppConfig":
-                "api_call_semaphore_limit": self.ai.api_call_semaphore_limit,
-                "api_call_max_retries": self.ai.api_call_max_retries,
-                "max_completion_tokens": self.ai.max_completion_tokens
+                "endpoint": self.endpoint,
+                "deployment_name": self.deployment_name,
+                "api_version": self.api_version,
+                "max_tokens": self.max_tokens,
+                "temperature": self.temperature,
+                "timeout": self.timeout,
+                "api_call_semaphore_limit": self.api_call_semaphore_limit,
+                "api_call_max_retries": self.api_call_max_retries,
+                "max_completion_tokens": self.max_completion_tokens
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)